skillmodels/tests/regression/test_model_no_stages_anchoring.json

```diff
@@ -16,7 +16,7 @@
                 ],
             "normalizations":
                 {
-                    "loadings": [{"y1": 1}, {"y1": 1}, {"y1": 1}, {"y1": 1},
+                    "loading": [{"y1": 1}, {"y1": 1}, {"y1": 1}, {"y1": 1},
```

Comment:
This was probably my mistake, but "loadings" should not be changed to "loading" in the model specification files. Otherwise I will have to adjust all my model specifications! Please revert the change in all model specification files and the corresponding functions in ModeSpecProcessor.

Reply:
I just saw that you did not make the corresponding changes in ModelSpecProcessor anyways. So right now the normalizations of "loading" would just be ignored. 

The check for invalid `norm_type` [here](https://github.com/janosg/skillmodels/blob/200aa1c301276cb20586223110f8df4c03c860c4/skillmodels/pre_processing/model_spec_processor.py#L230) does not work because [here](https://github.com/janosg/skillmodels/blob/200aa1c301276cb20586223110f8df4c03c860c4/skillmodels/pre_processing/model_spec_processor.py#L296) you already restrict the `norm_type` to valid values. 

Please correct this and make sure that a user would get an error if he specified "loading" in normalizations.

Reply:
done
and see [here](https://github.com/janosg/skillmodels/blob/1da49529564e4dac7cd9a0a7804845a0a91c54b9/skillmodels/pre_processing/model_spec_processor.py#L280)